stdlib/effect.mli

```diff
@@ -24,6 +24,9 @@ external perform : 'a t -> 'a = "%perform"
 
     @raise Unhandled if there is no active handler. *)
 
+external fiber_id : unit -> int64 = "caml_fiber_id"
+(** [fiber_id] returns an unique identifier for the current fiber. *)
```

Comment:
```suggestion
(** [fiber_id] returns a unique identifier for the current fiber. *)
```
("unique" is pronounced as if it started with "you", so "an" would sound wrong there.)